src/components/atoms/Dots.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="dots">
```

Comment:
Think if this atom should only contain one div with dots, but it's your call

Reply:
I wanted to keep it this way purely to make alignement easier; as rn it's intented to be in the middle of scoreboard categories.